ci/docker-build.sh

```diff
@@ -18,13 +17,7 @@ set -x
 
 # Build the base image which the image below can used.
 docker build \
-  --file ci/docker/Dockerfile.common \
-  --tag wasi-sdk-builder-base \
-  ci/docker
-
-# Build the container that is going to be used
-docker build \
-  --file ci/docker/Dockerfile.$1 \
+  --file ci/docker/Dockerfile \
   --tag wasi-sdk-builder \
   ci/docker
```

Comment:
With `--file` removed I think this whole command can now be on single line?

Also, do we need the `--tag`?